src/core/keybindings.c

```diff
@@ -198,12 +198,9 @@ reload_modmap (MetaDisplay *display)
             {
               if (syms[j] != 0)
                 {
-                  const char *str;
-
-                  str = XKeysymToString (syms[j]);
                   meta_topic (META_DEBUG_KEYBINDINGS,
                               "Keysym %s bound to modifier 0x%x\n",
-                              str ? str : "none",
+                               XKeysymToString (syms[j]) ? XKeysymToString (syms[j]) : "none",
```

Comment:
why? this seems less optimized, and less safe (it probably won't happen, but imagine XKeysymToString() not giving the same result on each call -- not sure if a keymap change could do that, but the point is it's not a great practice to this unless you know the call is trivial and constant)